index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+
+    <div id="main">
+        <div id="title">
+            <h1>Dr. Norman Borlaug</h1>
+        </div>
+
+        <div id="img-div">
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
```

Comment:
- [ ]  Remember that IMGs  :framed_picture: should always have an **alt** property. This way, if the img is broken, visitors can still understand what's the picture about! :hearts: Also, this is useful for indexation on search engines! 🌐